build.gradle

```
@@ -20,7 +20,7 @@ repositories {
 }
 
 archivesBaseName = "libblockatttributes"
-version = "0.8.4"
+version = "0.8.5-pre.1"
```

Versioning is likely going to be a massive pain, as I'll probably want to release 0.8.5 for 1.16.x at some point. Maybe this could use "0.8.1700" instead? (Mostly just to avoid confusion with the other versions).
```suggestion
version = "0.8.1700-pre.1"
```